.travis.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Please read section ``9.1.3`` about bare documents (just after your section about document markers). this is what is used in this file.